src/modules/likes.js

```diff
@@ -0,0 +1,32 @@
+// Async function to get likes for a show
+export const getLikes = async () => {
+  await fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/SzueLQl3FaQn1wJjDbsx/likes',
+  )
+    .then((response) => response.json())
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] To make your code clearn, kindly remove `eslint-disabler` from your project! 

> Suggestion: At that point. You can fix the issue by putting getLikes function after renderLikes. Then your eslint issue will be fixed